composer.json

```diff
@@ -16,21 +18,24 @@
             "homepage": "http://wyrihaximus.net/"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/inoas/twig-extensions"
+        }
+    ],
     "require": {
-        "php": "^5.5.9 || ^7.0",
-        "ajgl/breakpoint-twig-extension": "^0.3.0",
-        "aptoma/twig-markdown": "^2.0",
+        "php": "^7.0",
+        "aptoma/twig-markdown": "^3.0",
         "asm89/twig-cache-extension": "^1.0",
-        "cakephp/cakephp": "^3.2",
-        "jasny/twig-extensions": "^1.0",
-        "twig/twig": "^1.18"
+        "jasny/twig-extensions": "dev-twig2 as 1.1.0",
+        "cakephp/cakephp": "^3.5.2",
```

Comment:
Let me know if we should relax this to ^3.2 if possible.

Reply:
Nah lets keep it at latest 👍